docs/source/_static/js/custom.js

```diff
@@ -58,6 +81,63 @@ function addGithubButton() {
     document.querySelector(".wy-side-nav-search .icon-home").insertAdjacentHTML('afterend', div);
 }
 
+function addVersionControl() {
+    // To grab the version currently in view, we parse the url
+    let parts = location.toString().split('/');
```

Comment:
This doesn't change either, best practice is to set it as a `const`
```suggestion
    const parts = location.toString().split('/');
```